### DISCUSSION TITLE
Client render hook gets called twice

### DISCUSSION THREAD

**Discussion by rivatove** [vikejs/vike#609] [Discussion]
Hey, I'm trying to debug an issue where the client render hook gets called twice. Which of course makes the page render twice.

This only happens when serving the static assets with vite preview. It does not happen when using the vite dev server.

I haven't yet been able to reproduce it on a fresh app.

Do any of you know where I should search for a potential issue? Thank you in advance!

**Comment by brillout** [comment]
I wouldn't know. I'm afraid a reproduction is needed.

**Reply by rivatove** [reply]
Okay, so this problem happens when the Vite's official [splitVendorChunkPlugin](https://vitejs.dev/guide/build.html#chunking-strategy) is used. Do you accept issues for such things or is this out of scope?

**Reply by brillout** [reply]
If you can provide a minimal Vite + VPS reproduction, then I'll have a look at it.
